cypress/e2e/spec.cy.js

```diff
@@ -0,0 +1,6 @@
+describe('template spec', () => {
+  it('passes', () => {
+    cy.visit('http://127.0.0.1:8080/dist/index')
```

Comment:
Does there need to be a `index.html` file checked in?

Reply:
No webpack generates that for us